### PR TITLE
V5.0.x update LICENSE

### PR DESCRIPTION
**Pull request by janjust**
bot:notacherrypick

### CODE REVIEW

**Review by awlauria — APPROVED** [review]

**Review by awlauria — APPROVED** [review]

**Comment by awlauria** [comment]
the license changes should probs go back to main. thanks!

File: LICENSE [hunk 1]
```diff
@@ -45,22 +45,22 @@ Copyright (c) 2016      ARM, Inc.  All rights reserved.
...
 Copyright (c) 2016-2018 Broadcom Limited.  All rights reserved.
-Copyright (c) 2011-2021 Fujitsu Limited.  All rights reserved.
+Copyright (c) 2011-2023 Fujitsu Limited.  All rights reserved.
 Copyright (c) 2014-2015 Hewlett-Packard Development Company, LP.  All
                         rights reserved.
-Copyright (c) 2013-2021 Research Organization for Information Science (RIST).
+Copyright (c) 2013-2023 Research Organization for Information Science (RIST).
                         All rights reserved.
-Copyright (c) 2017-2021 Amazon.com, Inc. or its affiliates.  All Rights
+Copyright (c)           Amazon.com, Inc. or its affiliates.  All Rights
```
Comment (rhc54) [inline LICENSE:55]: Ummm...why are there no dates associated with Amazon any more? And did HPE, Fujitsu, Sandia, and Google actually submit changes this year?

Comment (janjust) [reply LICENSE:55]: I followed #9039 by bumping all '21 to '23. Having said that, v5.0 was in development well over 2 years I don't see how I can track down who all actually contributed. I could leave it out apart from the key contributors that I'm aware of and let others complain. I figured this was safer. As far as Amazon, they're going to a dateless format and asked me to remove the dates.
I see that Nanuk is 2021 - does it need an end date or update to '23?

**Review by lrbison — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (rhc54) [reply LICENSE:55]: > I see that Nanuk is 2021 - does it need an end date or update to '23?

Doesn't really matter - I have no intentions of enforcing anything anyway

> As far as Amazon, they're going to a dateless format and asked me to remove the dates.

Well, based on what our lawyers told us when we started the project, a copyright without date is legally meaningless. You cannot claim a time-unlimited copyright - no such legal thing. Don't know if it invalidates the entire license, or if it just means Amazon has no legal rights. You should check with your own lawyers to see before releasing it that way.

Comment (janjust) [reply LICENSE:55]: I've spoken to amazon guys, readding the date is fine too, so for consistency I'll do revert their dates.

Comment (janjust) [reply LICENSE]: @rhc54 done

**Review by lrbison — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ application developers and computer science researchers.
 
 The Open MPI documentation can be viewed in the following ways:
 
-1. Online at https://docs.open-mpi.org/
+1. Online at https://docs.open-mpi.org/en/v5.0.0/
```
Comment (jsquyres) [inline README.md:17]: I would not mention the `en/v5.0.0/` link here.  If you surf to https://docs.open-mpi.org/, you'll get redirected to the latest docs.  If you want to see a different version, you can use the navigation picker to pick the specific version you want.

If you list `v5.0.0` here, you'll have to update this link every time you do a new release.

**Review by wenduwan — APPROVED** [review]
Thank you Tommy!

File: README.md (hunk 2 above)
Comment (janjust) [reply README.md]: done, I dropped the commit

**Review by jsquyres — APPROVED** [review]

**Pull request merged**